spec/support/register_helpers.rb

```diff
@@ -0,0 +1,11 @@
+module RegisterHelpers
+  def set_registerable_option(value)
+    Spree::Auth::Config[:registerable] = value
+    Spree.send(:remove_const, 'User')
+    load File.expand_path("../../../app/models/spree/user.rb", __FILE__)
```

Comment:
I know this was copied from elsewhere in the project and this project doesn't appear to enforce consistent quote styles, but we should still try to be consistent:

```suggestion
    load File.expand_path('../../../app/models/spree/user.rb', __FILE__)
```